invenio_app_ils/mail/messages.py

```diff
@@ -59,7 +71,7 @@ def __init__(
         except TemplateError:
             kwargs["html"] = kwargs["body"]
 
-        footer_tmpl = current_app.jinja_env.get_template(footer_template)
+        footer_tmpl = current_app.jinja_env.get_template(footer_value)
```

Comment:
it looks to me a little too complex maybe?
We could define in the config:
```
ILS_GLOBAL_MAIL_TEMPLATES = {
    "footer": "invenio_app_ils/mail/footer.html"
}
```

and then when using it, simply:
```
class BlockTemplatedMessage(Message):

    def __init__(
        self,
        template,
        ctx={},
        **kwargs
    ):

    ...
    footer_template = current_app.config.get("ILS_GLOBAL_MAIL_TEMPLATES", {}).get("footer)
    if footer_template:
        ...
        footer_tmpl = current_app.jinja_env.get_template(footer_template)
        ...
```
and not declare any default in this class